### DISCUSSION TITLE
Pizza pics

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/feedback#33] [Show and tell]
Drop an image of a pizza.

**Comment by nickytonline** [comment]
A pepperoni pizza from Chicago Pizza, http://chicagopizzamontreal.ca/

![A pepperoni pizza from Chicago Pizza](https://github.com/open-sauced/community/assets/833231/5781267d-a330-45cc-85d5-266a5b5b893a)

**Comment by bdougie** [comment]
![image](https://github.com/open-sauced/community/assets/5713670/e44b57f7-3a95-4be9-905f-9673d6301f15)

Best pizza in North Oakland. [Arthur Macs](https://arthurmacs.com)

**Comment by jpmcb** [comment]
![IMG_4087](https://github.com/open-sauced/community/assets/23109390/cd7f46e9-849f-47a7-83de-6f429a822cdf)

Late night slices at [Prince St Pizza](https://maps.app.goo.gl/TLQrd7qKhxCw9RbC9), one of my favorite places in Manhattan

**Comment by ryanfurrer** [comment]
Plain and pepperoni slices from [Mela Pizza](https://www.melapizzamenu.com/) in Astora, Queens, NY

![Plain and pepperoni pizza slices](https://github.com/open-sauced/community/assets/40150036/b476c8f8-5b51-4984-a380-833e1d507ee5)

**Reply by FatumaA** [reply]
Tried the local Pappa John's and was underwhelmed so today it's homemade pizza 😊

![20240609_181842](https://github.com/open-sauced/community/assets/67555014/966160aa-60d4-4506-a482-df596edb26fe)

**Comment by DominicAsio32** [comment]
Enjoy a some delicious pies at a Italian San Jose Restaurant called Eataly - https://www.eataly.com/us_en/stores/silicon-valley
![IMG_4253](https://github.com/open-sauced/community/assets/94804174/41489f88-823f-4b9e-ac34-db111992228b)

**Comment by Kratosgado** [comment]
I need to taste a correct Pizza

**Comment by shaundai** [comment]
Oxtail Pizza from [Grand Slam Pizza](https://www.gspofkennesaw.com/) of Kennesaw, GA!
![oxtailpizza](https://github.com/user-attachments/assets/7cf8a018-0c5c-410e-b97e-a76531fd2ff3)

**Comment by chrislanejones** [comment]
Pineapple pizza for [That Conference](https://thatconference.com/wi/2024) Pineapple pizza haters - UNO Pizzeria & Grill - Lake Delton, Wisconsin

![GT6063sWUAAskah](https://github.com/user-attachments/assets/948e412c-3815-43bd-b9a3-7ecd7ef7943e)
